src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
I had that originally (the `components` namespace object), and everybody bitched about it constantly, so I'll say no. Apparently it's better to have one giant noisy namespace for everything.